finetune_instruct_pix2pix.py

```diff
@@ -1075,7 +1082,7 @@ def collate_fn(examples):
                 pipeline.set_progress_bar_config(disable=True)
 
                 # run inference
-                original_image = download_image(args.val_image_url)
+                load_image(args.val_image_url)
```

Comment:
`load_image()` doesn't return anything and is neither a closure inside `main()` so, I am sure about the effectivity of this.